example-ts/index.ts

```diff
@@ -0,0 +1,228 @@
+import * as THREE from 'three'
+import { THREEx, ARjs } from "@ar-js-org/ar.js-threejs"
+import { log } from 'console';
+
+THREEx.ArToolkitContext.baseURL = "./";
+// log in the console the two newly created namespaces
+console.log(THREEx);
+console.log(ARjs)
+//////////////////////////////////////////////////////////////////////////////////
+//		Init
+//////////////////////////////////////////////////////////////////////////////////
+
+// init renderer
+var renderer = new THREE.WebGLRenderer({
+    antialias: true,
+    alpha: true
+});
+renderer.setClearColor(new THREE.Color('lightgrey'), 0)
+renderer.setSize(640, 480);
+renderer.domElement.style.position = 'absolute'
+renderer.domElement.style.top = '0px'
+renderer.domElement.style.left = '0px'
+document.body.appendChild(renderer.domElement);
+
+// array of functions for the rendering loop
+var onRenderFcts: any[] = [];
+var arToolkitContext: any, arMarkerControls;
+
+// init scene and camera
+var scene = new THREE.Scene();
+
+//////////////////////////////////////////////////////////////////////////////////
+//		Initialize a basic camera
+//////////////////////////////////////////////////////////////////////////////////
+
+// Create a camera
+var camera = new THREE.Camera();
+scene.add(camera);
+
+
+var arToolkitSource = new THREEx.ArToolkitSource({
+    // to read from the webcam
+    sourceType: 'webcam',
+
+    sourceWidth: window.innerWidth > window.innerHeight ? 640 : 480,
+    sourceHeight: window.innerWidth > window.innerHeight ? 480 : 640,
+
+    sourceUrl: '',
+
+    displayWidth: 640,
+    displayHeight: 480,
+    
+    deviceId: ''
+})
+
+arToolkitSource.init(function onReady() {
+    console.log('here');
+    
+    arToolkitSource.domElement.addEventListener('canplay', () => {
+        console.log(
+            'canplay',
+            'actual source dimensions',
+            //arToolkitSource.domElement.videoWidth,
```

Comment:
i can not add this because:
```
Property 'videoWidth' does not exist on type 'HTMLImageElement | HTMLVideoElement'.
  Property 'videoWidth' does not exist on type 'HTMLImageElement'.ts(2339)
```

Reply:
i declared `domELement `as `any`, not properly solved but it's fine for now. I will open an issue for this.